src/backlight/metrics/trade_metrics.py

```diff
@@ -18,15 +18,24 @@ def _sum(a: pd.Series) -> float:
 
 
 def _calc_pl(trade: Trade, mkt: MarketData) -> float:
-    assert trade.index.isin(mkt.index).all()
     mkt = mkt.loc[trade.index, :]
-
     positions = backlight.positions.calc_positions((trade,), mkt)
     pl = calc_pl(positions)
     return _sum(pl)
 
 
 def count_trades(trades: Trades, mkt: MarketData) -> Tuple[int, int, int]:
+    """ Count total trades, win trades and lose trades
+
+    Args:
+        trades : Trades to be evaluated. Each trade is evaluated
+                 only if it contains more than one transactions,
+                 because we can define pl in that case.
+        mkt: Market data. The index should contains all trades' index.
+
+    Returns:
+        total count, wind count, lose count
```

Comment:
```suggestion
        total count, win count, lose count
```
Sail with the wind! 💨 ⛵️